pymagicc/io.py

```diff
@@ -2864,116 +2516,363 @@ def rename_col(x):
             output, index=False, header=False, formatters=formatters, sparsify=False
         )
         output.write(self._newline_char)
+
         return output
 
-    def _write_variable_datablock_concentrations(self, output, data_block, units_level):
-        col_order = [
-            "VARIABLE",
-            "CO2EQ_CONC",
-            "KYOTOCO2EQ_CONC",
-            "CO2_CONC",
-            "CH4_CONC",
-            "N2O_CONC",
-            "FGASSUMHFC134AEQ_CONC",
-            "MHALOSUMCFC12EQ_CONC",
-            "CF4_CONC",
-            "C2F6_CONC",
-            "C6F14_CONC",
-            "HFC23_CONC",
-            "HFC32_CONC",
-            "HFC4310_CONC",
-            "HFC125_CONC",
-            "HFC134A_CONC",
-            "HFC143A_CONC",
-            "HFC227EA_CONC",
-            "HFC245FA_CONC",
-            "SF6_CONC",
-            "CFC11_CONC",
-            "CFC12_CONC",
-            "CFC113_CONC",
-            "CFC114_CONC",
-            "CFC115_CONC",
-            "CCL4_CONC",
-            "CH3CCL3_CONC",
-            "HCFC22_CONC",
-            "HCFC141B_CONC",
-            "HCFC142B_CONC",
-            "HALON1211_CONC",
-            "HALON1202_CONC",
-            "HALON1301_CONC",
-            "HALON2402_CONC",
-            "CH3BR_CONC",
-            "CH3CL_CONC",
-        ]
+    def _get_col_order_rename_col(self):
+        if self._filepath.endswith("_RADFORCING.DAT"):
```

Comment:
Are the following the same except for the _RF/_EFFRF?

Reply:
resolved by e5792b74b7bc7727031ffa4a1d54c0adc2d8cd96